src/api/authenticated/orders/submit_order/mod.rs

```diff
@@ -115,9 +115,9 @@ impl<'a> Endpoint for SubmitOrder<'a> {
 pub struct SubmitOrderResp {
     pub mts: u64,
     pub ty: String,
-    pub message_id: u64,
-    pub order: Order,
-    pub code: u64,
+    pub message_id: Option<u64>,
+    pub orders: Vec<Order>,
```

Comment:
I simplified this to only keep one order. As the do is saying [here](https://docs.bitfinex.com/reference/rest-auth-submit-order), the returned orders array will only contain the new submitted order.